### PR TITLE
[docs]: translate Custimize Error Handler

### PR DESCRIPTION
**Pull request by rogerogers**
#### What type of PR is this?

docs: translate Custimize Error Handler

#### What this PR does / why we need it (en: English/zh: Chinese):

en:  translate Custimize Error Handler
zh:  翻译错误处理

#### Which issue(s) this PR fixes:

https://github.com/cloudwego/kitex/issues/569

### CODE REVIEW

**Comment by rogerogers** [comment]
https://github.com/cloudwego/cloudwego.github.io/blob/f8a38ab5f6a546009a1074d9c40741f4b7997aed/content/zh/docs/kitex/Tutorials/advanced-feature/error_handler.md?plain=1#L21
在 RPC 层面其实是请求成功，这里是不是应该是在RPC层面可能是请求成功。

File: content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md [hunk 1]
```diff
@@ -1,8 +1,92 @@
 ---
 title: "Customize Error Handler"
-date: 2022-07-20
+date: 2022-07-22
 weight: 4
 description: >
 ---
 
-Translation TODO
+Some users will ask how to let the client-side receive the corresponding error type of the server-side, here to explain, RPC communicates through the protocol, and error handling is also based on the protocol. usually, server-side returns Error, RPC framework unified error coding will return to the client, if you want the client to return the same error as the server-side, you need to define a set of error codes for handling. However, considering that RPC does not have a unified error code specification, and internal error codes are not necessarily applicable to external users, so the open source part of Kitex does not expose the error code definition, and users can customize their own error handler by providing **ErrorHandler**.
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md:8]: Usually when the server returns Error, the framework will unify the Error encoding and return it to the client side.
... by using the provided **ErrorHandler**

Comment (rogerogers) [reply content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md:8]: > Usually when the server returns Error, the framework will unify the Error encoding and return it to the client side. ... by using the provided **ErrorHandler**

用户可以通过提供 ErrorHandler 来定制自己的错误处理。

users can customize their own error handler by using the provided ErrorHandler,

这个翻译过来有点对不上，用户可以通过提供 ~~的~~ ErrorHandler来定制自己的错误处理。是中文少了个的吗

Comment (GuangmingLuo) [reply content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md:8]: 嗯，这个其实可以去理解下上下文，有时间中文表达会比较模糊欠精准。框架提供的是接口和配置入口，具体由用户提供实现。

Comment (rogerogers) [reply content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md:8]: 了解。

Comment (rogerogers) [reply content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md]: fixed.

File: content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md [hunk 2]
```diff
@@ -1,8 +1,92 @@
...
 ---
 
-Translation TODO
+Some users will ask how to let the client-side receive the corresponding error type of the server-side, here to explain, RPC communicates through the protocol, and error handling is also based on the protocol. Usually when the server returns Error, the framework will unify the Error encoding and return it to the client side. If you want the client to return the same error as the server-side, you need to define a set of error codes for handling. However, considering that RPC does not have a unified error code specification, and internal error codes are not necessarily applicable to external users, so the open source part of Kitex does not expose the error code definition, and users can customize their own error handler by using the provided **ErrorHandler**.
+
+## Recommended Usage
+
+ErrorHandler is configured via the client/server Option, but usually, a microservice system will have a unified error handler specification, if you are an enterprise user, it is recommended to customize the Option through the [Suite](...). /... /framework-exten/suite), so that the service developer does not need to pay attention to the configuration of error handler.
```
Comment (GuangmingLuo) [inline content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md:12]: `[Suite](../../framework-exten/suite) `

Comment (rogerogers) [reply content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md]: sorry, fixed it and tried, link works fine now.

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**